website-nextjs/src/pages/dashboard/performance-history.tsx

```diff
@@ -186,6 +186,22 @@ const PagePerformanceHistory = () => {
                 automatically re-generate to show you the performance history on
                 your chosen subset.
               </div>
+              <div className="font-lato font-normal/1.4 text-l max-w-screen-lg">
+                We run the latest version of each solver released on conda in
+                that year. 2025's versions were selected if the solver had a
```

Comment:
```suggestion
                that year. 2025&apos;s versions were selected if the solver had a
```
@jacek-oet is this the right way to fix this error?
```
191:32  Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.  react/no-unescaped-entities
```

Reply:
Yes, it is okay.